### PR TITLE
[TH2-2189] Add ability to load secrets from a file

### PR DESCRIPTION
**Pull request by ivandruzhinin**

### CODE REVIEW

File: src/main/kotlin/com/exactpro/th2/common/schema/configuration/Secrets.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.exactpro.th2.common.schema.configuration
```
Comment (OptimumOpium) [inline src/main/kotlin/com/exactpro/th2/common/schema/configuration/Secrets.kt:1]: Missing copyright

**Review by OptimumOpium — COMMENTED** [review]
As far as I remember there should be an option to load the box configuration from the k8s. Do we need to add the secrets support for that mode as well?

**Comment by ivandruzhinin** [comment]
> As far as I remember there should be an option to load the box configuration from the k8s. Do we need to add the secrets support for that mode as well?

I think this would handle that - https://github.com/th2-net/th2-common-j/commit/f762ee88b0593ec9864cc48d01c879dc058abc11#diff-6313585584b95b2851b150bc62e0bb207697681044acd9c28c1afa25246daf1dR480

**Comment by OptimumOpium** [comment]
> > As far as I remember there should be an option to load the box configuration from the k8s. Do we need to add the secrets support for that mode as well?
> 
> I think this would handle that - [f762ee8#diff-6313585584b95b2851b150bc62e0bb207697681044acd9c28c1afa25246daf1dR480](https://github.com/th2-net/th2-common-j/commit/f762ee88b0593ec9864cc48d01c879dc058abc11#diff-6313585584b95b2851b150bc62e0bb207697681044acd9c28c1afa25246daf1dR480)

Yes, you are probably right. Missed that

**Pull request closed without merging**